manifests/d.pp

```diff
@@ -80,6 +86,7 @@
   $staleness_check_params=undef,
   $lock=false,
   $timeout=undef,
+  $timeout_custom_args=undef,
```

Comment:
Can we just make this $timeout_signal_arg = '-s 9' ?

If not, false is preferred to undef, unless you're explicitly planning to use the special behaviour of undef, which I don't think you are.